creator-node/nginx_conf/nginx.conf

```diff
@@ -63,10 +63,27 @@ http {
             add_header X-Cache-Status $upstream_cache_status always;
         }
 
+        location = /health_check {
+            proxy_cache cache;
+
+            proxy_pass http://127.0.0.1:3000;
+
+            # Set client IP as `X-Forwarded-For` response header
+            proxy_set_header X-Forwarded-For $remote_addr;
+
+            # Never serve a stale response
+
+            # Cache responses with 200 status, for 1 second
+            proxy_cache_valid 200 1s;
```

Comment:
i intentionally didn't add that since it didn't seem to have any value, and since others could then DOS us by using this bypass, but i guess thats no worse than our status quo so fair enough, i'll add it back

Reply:
done